verticapy/performance/vertica/qprof.py

```diff
@@ -1662,6 +1662,47 @@ def get_qplan_profile(
                 **style_kwargs,
             )
         return vdf
+    
+    # Step 10: Query events
+    def get_query_events(self) -> vDataFrame:
+        """
+        Returns a vDataFrame that contains a table listing query events
+
+        Parameters
```

Comment:
when we don't have parameters, we skip the section.

Reply:
Skipped this section